docs/known_issues.md

```diff
@@ -14,15 +14,24 @@ As a workaround, use `tag.turbo_frame` instead of `turbo_frame_tag`.
 
 Note: For the same functionality as `turbo_frame_tag(my_model)`, use `tag.turbo_frame(id: dom_id(my_model))`.
 
-## form_for compatibility
+## Compatibility with Rails form helpers
 
 ViewComponent [isn't compatible](https://github.com/viewcomponent/view_component/issues/241) with `form_for` helpers by default.
 
-## Inconsistent controller rendering behavior between Rails versions
+Passing a form object reference (usually `f`) to a view component works for simple cases, like `f.text_field :name`.
+Content will be ill-ordered or duplicated in complex cases (such as passing blocks to form helpers or when nesting components).
 
-In versions of Rails < 6.1, rendering a ViewComponent from a controller doesn't include the layout.
+Consider following options:
 
-## Forms don't use the default form builder
+- Render a form end-to-end within one view component
+- Render a [classical view partial](https://guides.rubyonrails.org/layouts_and_rendering.html#using-partials) within the view component, which includes the form
+- Use `[FormBuilder](https://guides.rubyonrails.org/form_helpers.html#customizing-form-builders)` to create reusable form components.
+  Note: There are multiple approaches how to implement a FormBuilder:
+  - Classic approach (without ViewComponent)
```

Comment:
Can you elaborate on what "classic approach" means? You mean not using ViewComponents at all to generate the form, right?

Reply:
in the sense of how to customize the form builder. I mean, whatever you find when you google for "custom form builder rails"